client/src/App.jsx

```diff
@@ -13,18 +13,25 @@ const withSuspense = (Component) => (
 
 const HomePage = React.lazy(() => import("./pages/HomePage"));
 const SearchResultsPage = React.lazy(() => import("./pages/SearchResultsPage"));
+const AboutPage = React.lazy(() => import("./pages/AboutPage"));
+
+export const AuthContext = createContext([{}, () => {}]);
```

Comment:
We should separate out contexts into a separate file, especially if we're going to be using this in other parts of the application

Reply:
I left a comment about the requirement for auth in this milestone, i suggest we park it unless we've decided we have time to hit some of the other dependent features as well